authentik/enterprise/public.pem

```diff
@@ -0,0 +1,10 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
You can get the production one from `https://vault.customers.goauthentik.io/v1/licensing-ca/ca/pem`